robolectric/src/test/java/org/robolectric/shadows/ShadowICUTest.java

```diff
@@ -1,22 +1,47 @@
 package org.robolectric.shadows;
 
+import android.app.Activity;
 import android.os.Build;
+import android.os.Bundle;
+import android.widget.DatePicker;
+import android.widget.LinearLayout;
 
 import libcore.icu.ICU;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
 import org.robolectric.TestRunners;
 import org.robolectric.annotation.Config;
+import org.robolectric.util.ActivityController;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(TestRunners.MultiApiWithDefaults.class)
 @Config(sdk = {
-    Build.VERSION_CODES.LOLLIPOP })
+  Build.VERSION_CODES.LOLLIPOP,
+  Build.VERSION_CODES.LOLLIPOP_MR1})
```

Comment:
I'm getting a compile error that M doesn't exist. The M sdk is installed on my computer (API 23), and I have tried to rerun install-dependencies.rb with the M sdk installed.

Do I need to do anything else, in order for Build.VERSION_CODES to generate the M entry in its class?